documents/Specification/MaterialX.Specification.md

```diff
@@ -53,6 +53,8 @@ This document describes the core MaterialX specification.  Companion documents [
  [Standard Source Nodes](#standard-source-nodes)  
   [Texture Nodes](#texture-nodes)  
   [Procedural Nodes](#procedural-nodes)  
+  [Noise Nodes](#noise-nodes)  
+  [Shape Nodes](#shape-nodes)  
```

Comment:
Should we make this change in tandem with updates to node groups in our data libraries, e.g. setting them to `shape` or `noise` as appropriate?

Reply:
We've had many instances where the spec description is "where we want to go", and I believe there was strong approval for grouping the noise and shape nodes under these specific new headings.  In order to move the actual nodes into new nodegroups, we'd need to resolve the situation where those nodegroup names are affecting codegen: some other "hint"-like attribute or something would need to be proposed as an alternative mechanism.